dev-tools/install.sh

```diff
@@ -28,7 +28,11 @@ echo λλλ phive
 if [ ! -x bin/phive ]; then
     wget -Obin/phive https://phar.io/releases/phive.phar
     wget -Obin/phive.asc https://phar.io/releases/phive.phar.asc
-    gpg --keyserver pool.sks-keyservers.net --recv-keys 0x9D8A98B29B2D5D79
+    gpg --list-keys 0x9D8A98B29B2D5D79 \
```

Comment:
For this first command to ever exit successfully, the key from a previous build must be present, but as far as I can tell this will never happen as the GPG folder is not in the cache list:

https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/45ec06c723d26243124a9cba84267dad13ed11dc/.travis.yml#L6-L10

Am I correct? If so, may we add `$HOME/.gnupg` to the list so the key persists across builds?

Reply:
You're correct that there will be no key available on Travis.

This is the reason why if the key does not exists `gpg --list-keys` returns non-zero (failure) so the next command in the chain will be executed (OR'ed by `||`).

This can be tested locally by deleting that public key in case it already exists (and also by removing `devtools/bin/phive` in case it exists) and then invoking the dev-tools-install script. This simulates the situation on Travis where the key does not exists by default.

Reply:
Ok, I get how this works, so: can we add `$HOME/.gnupg` to the cache folders list? Otherwise `gpg --list-keys 0x9D8A98B29B2D5D79` is just useless

Reply:
The script is portable and idempotent, so I would not call it out useless. About that cache folder I don't know whether or not, I don't have an opinion about that.

Reply:
I would have a try without cache and add it later if that can improve things much.